.devcontainer/Dockerfile

```diff
@@ -1,26 +1,30 @@
-FROM mcr.microsoft.com/devcontainers/base:ubuntu
-COPY environment.yml /opt/environment.yml
-
-# Enable by default the conda environment for all users.
-# The prefix is the one used by the micromamba feature.
-ENV CONDA_PREFIX_JAXSIM=/opt/conda/envs/jaxsim
-RUN echo 'function activate_conda() {' >> /etc/bash.bashrc &&\
-    echo '  eval "$(micromamba shell hook -s bash)"' >> /etc/bash.bashrc &&\
-    echo '  micromamba activate ${CONDA_PREFIX_JAXSIM-$CONDA_PREFIX}' >> /etc/bash.bashrc &&\
-    echo '}' >> /etc/bash.bashrc &&\
-    echo '[[ -x $(which micromamba) && -d ${CONDA_PREFIX_JAXSIM-$CONDA_PREFIX} ]] && activate_conda' >> /etc/bash.bashrc &&\
-    echo '[[ -x $(which micromamba) && ! -x $(which mamba) ]] && alias mamba="$(which micromamba)"' >> /etc/bash.bashrc &&\
-    echo '[[ -x $(which micromamba) && ! -x $(which conda) ]] && alias conda="$(which micromamba)"' >> /etc/bash.bashrc
-
-# Provide libGL.so.1 from the host OS
-RUN sudo apt-get update &&\
-    sudo apt-get install -y --no-install-recommends libgl1 &&\
-    rm -rf /var/lib/apt/lists/*
-
-# The Python extension in VSCode is not able to detect the interpreter installed with micromamba.
-# As documented as follows, we provide a suggestion through an environment variable.
-# https://code.visualstudio.com/docs/python/environments#_where-the-extension-looks-for-environments
-ENV WORKON_HOME=$CONDA_PREFIX_JAXSIM
-
-# Specify the workdir of the container
+FROM ghcr.io/prefix-dev/pixi:latest
+
+COPY . /workspace/jaxsim
 WORKDIR /workspace/jaxsim
+
+# Add Git LFS package repository and install
+RUN apt-get update && apt-get install -y curl \
+    && curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash \
+    && apt-get install -y git-lfs
+
+# Initialize Git LFS
+RUN git lfs install
+
+# Pull the `pixi.lock` file from LFS
+RUN git-lfs pull --include="pixi.lock"
+
+# Install the dependencies
+RUN if [ ! -d ".pixi/envs/" ]; then pixi install --environment test-cpu; fi
+
+# Create the shell-hook bash script to activate the environment
+RUN pixi shell-hook --environment test-cpu > /shell-hook.sh
+
+# Extend the shell-hook script to run the command passed to the container
+RUN echo 'exec "$@"' >> /shell-hook.sh
+
+# Make the shell-hook script executable
+RUN chmod +x /shell-hook.sh
```

Comment:
Suggestions: to keep docker images slim, It's advisable using as many layers as possible (it means merging together commands that might belong to the same group).

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/276/commits/831046e164691ee7861dd38c0afb0be56b7378c8

Reply:
Did this get lost in a rebase?

Reply:
Yes, thanks for noticing it! Done in https://github.com/ami-iit/jaxsim/pull/276/commits/0c5395d933a6b9052ca09f1f727db0edea6e4bb7